packages/ckeditor5-source-editing/theme/sourceediting.css

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+@import "../../ckeditor5-theme-lark/theme/mixins/_rounded.css";
+@import "../../ckeditor5-theme-lark/theme/mixins/_focus.css";
+@import "../../ckeditor5-theme-lark/theme/mixins/_shadow.css";
```

Comment:
Shouldn't it be `@ckeditor/ckeditor5-theme-lark`?

Reply:
Import path changed. Moved `@ckeditor/ckeditor5-theme-lark` from `devDependencies` to `dependencies`.